packages/kbn-doc-links/src/get_doc_links.ts

```diff
@@ -34,6 +34,8 @@ export const getDocLinks = ({ kibanaBranch }: GetDocLinkOptions): DocLinks => {
   const WORKPLACE_SEARCH_DOCS = `${ELASTIC_WEBSITE_URL}guide/en/workplace-search/${DOC_LINK_VERSION}/`;
   const SEARCH_UI_DOCS = `${DOCS_WEBSITE_URL}search-ui/`;
   const MACHINE_LEARNING_DOCS = `${ELASTIC_WEBSITE_URL}guide/en/machine-learning/${DOC_LINK_VERSION}/`;
+  const SERVERLESS_DOCS = `${DOCS_WEBSITE_URL}serverless/`;
```

Comment:
We do not yet have a model for the Serverless docs URLs, but it looks like this would point to https://www.elastic.co/guide/serverless. 

The Serverless docs will be hosted on docs.elastic.co.

Reply:
`DOCS_WEBSITE_URL` evaluates to `https://docs.elastic.co/` (see Search UI definition a few lines above). So I think this PR has the URLs correct. I'm going to approve.